rke/k8s_rke_system_images.go

```diff
@@ -5604,6 +5647,49 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/kubelet-pause:v0.1.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.15.13",
 		},
+		// Enabled in v2.5.10
```

Comment:
Due to the OOB release and not waiting for v2.5.10, this comment will need to be `Enabled out of band post v2.5.9`

Reply:
Updated. Please check.